console_bridge library not found when chaining workspace using binaries

When using the binaries we produce for Linux and when building a workspace chained off the workspace provided by those binaries, the `components` package (and perhaps others) fail to find `console_bridge`'s library generating this CMake error:

```
CMake Error at /home/william/testing/nightly/ros2-linux/lib/console_bridge/cmake/console_bridge-config.cmake:15 (message):
  Library 'console_bridge' in package console_bridge is not installed
  properly
Call Stack (most recent call first):
  /home/william/testing/nightly/ros2-linux/share/class_loader/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package)
  /home/william/testing/nightly/ros2-linux/share/class_loader/cmake/class_loaderConfig.cmake:30 (include)
  CMakeLists.txt:11 (find_package)

```

I used the most recent binary package from our packaging job ([job-#569](http://ci.ros2.org/view/packaging/job/packaging_linux/569/)) on the build farm to reproduce this, but it was originally reported in https://github.com/ros2/ros2/issues/309 with the binaries for `release-beta1`. These are the steps to reproduce the issue:

- `mkdir -p /tmp/nightly/demos_ws/src`
- `cd /tmp/nightly`
- `wget http://ci.ros2.org/view/packaging/job/packaging_linux/lastSuccessfulBuild/artifact/ws/ros2-package-linux.tar.bz2`
- extract `ros2-package-linux.tar.bz2` to the current directory
- `source ./ros2-linux/setup.zsh`
- `cd ./demos_ws/src`
- `git clone https://github.com/ros2/demos.git`
- `cd ..`
- `ament build`

At this point it fails to build with the above error. I also tried with a different build type:

- `rm -rf build`
- `ament build --cmake-args -DCMAKE_BUILD_TYPE=Release --`

This also fails.

--- a/ros2.repos
+++ b/ros2.repos
@@ -51,6 +51,10 @@ repositories:
     type: git
     url: https://github.com/ros/class_loader.git
     version: ros2
+  ros/console_bridge:
+    type: git
+    url: https://github.com/ros/console_bridge.git
+    version: master
   ros2/ament_cmake_ros:
     type: git
     url: https://github.com/ros2/ament_cmake_ros.git
@@ -195,7 +199,3 @@ repositories:
     type: git
     url: https://github.com/ros2/vision_opencv.git
     version: ros2
-  vendor/console_bridge:
-    type: git
-    url: https://github.com/ros/console_bridge.git
-    version: master

--- a/ros2.repos
+++ b/ros2.repos
@@ -59,10 +59,6 @@ repositories:
     type: git
     url: https://github.com/ros2/common_interfaces.git
     version: master
-  ros2/console_bridge:
-    type: git
-    url: https://github.com/ros2/console_bridge.git
-    version: ros2
   ros2/demos:
     type: git
     url: https://github.com/ros2/demos.git
@@ -199,3 +195,7 @@ repositories:
     type: git
     url: https://github.com/ros2/vision_opencv.git
     version: ros2
+  vendor/console_bridge:
+    type: git
+    url: https://github.com/ros/console_bridge.git
+    version: master

